### PR TITLE
Slackbot:release

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

- Adds `lg slackbot release` to main CLI
- Adds tests to slackbot
- Support for other repos & scopes

### CODE REVIEW

File: tools/slackbot/src/release-bot/release-bot.types.ts [hunk 1]
```diff
@@ -4,7 +4,9 @@ export const Channels = {
   'leafygreen-ui-releases': 'C01CBLPFD35',
 } as const;
 
-export interface Opts {
+export type ChangeType = 'major' | 'minor' | 'patch' | 'dependency';
```
Comment (shaneeza) [inline tools/slackbot/src/release-bot/release-bot.types.ts:7]: Is the `dependency` type new?

File: tools/slackbot/src/release-bot/utils/utils.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+/* UTILS */
+
+export function exists<T extends string | Array<any>>(
+  arg: T | undefined,
+): arg is T {
+  return !isUndefined(arg) && arg.length > 0;
+}
+
+export function isValidJSON(str?: string): boolean {
```
Comment (shaneeza) [inline tools/slackbot/src/release-bot/utils/utils.ts:11]: Is there a way to share this? It looks like you're also using a few lines above.

**Review by shaneeza — APPROVED** [review]

File: tools/slackbot/src/release-bot/release-bot.types.ts (hunk 1 above)
Comment (TheSonOfThomp) [reply tools/slackbot/src/release-bot/release-bot.types.ts:7]: It's just a special designation we have we want to format them differently. `dependency` change type is just a patch with nothing other than dependency bumps

File: tools/slackbot/src/release-bot/utils/utils.ts (hunk 2 above)
Comment (TheSonOfThomp) [reply tools/slackbot/src/release-bot/utils/utils.ts:11]: @shaneeza What do you mean share? Like add it to lib/meta?

Comment (shaneeza) [reply tools/slackbot/src/release-bot/utils/utils.ts:11]: Yeah, or another appropriate place. Cause right now, you're writing this function in both `tools/slackbot` and `tools/meta`

Comment (TheSonOfThomp) [reply tools/slackbot/src/release-bot/utils/utils.ts]: 👍

**Pull request merged**